### PR TITLE
Update gulp to version 4.0.0

### PR DESCRIPTION
**Pull request by LMojica**
Updates gulp to version 4.0.0 to allow Travis builds to run on Node 12.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const babel = require('gulp-babel');
 const del = require('del');
 const gulp = require('gulp');
 const eslint = require('gulp-eslint');
-const replace = require("gulp-replace");
-const uglify = require("gulp-uglify");
-const packageJson = require("./package.json");
+const replace = require('gulp-replace');
+const uglify = require('gulp-uglify');
+const packageJson = require('./package.json');
```
Comment (gigabo) [inline gulpfile.babel.js:7]: It is a little hard to pick out the relevant changes in this diff among all of the quote-style changes.

These are not related to the upgrade, right?

Comment (LMojica) [reply gulpfile.babel.js:7]: Right. The file used a mix of double-quoted strings and single-quoted strings. I just converted all of them to single-quoted strings.

Comment (gigabo) [reply gulpfile.babel.js:7]: Minimal, targeted diffs generally make life easier for reviewers who are looking at your code.

I like that you want to clean the quote situation up in here, but in the future please do this sort of thing in a dedicated patch.
